package.json

```diff
@@ -17,6 +17,8 @@
   "scripts": {
     "lint": "eslint src --ext .js --ext .jsx",
     "start": "webpack-dev-server --host 0.0.0.0 --port 8080 --hot --progress",
+    "start-httpserver": "http-server .",
```

Comment:
`serve` is shorter, and shouldn't this *serve* the `dist` directory?

Reply:
> `serve` is shorter

Will update.

> shouldn't this serve the dist directory

That's what the author did and I just moved it here from the shell script. Btw why should it serve the `dist` directory? `index.html` is in the project root directory.